### DISCUSSION TITLE
Feature: Set technical contact on multiple projects at once or default technical contact

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Cloud.Issues#810] [Ideas]
### Issue description

Often developers are creating Umbraco Cloud projects, and by default it set that person as technical contact. But that is not always what we want.

Typical it is project manager or tech lead in organization, who should be technical contact on a project.

We can change that on each project, but as the number of projects in an organization grows, it's getting harder to manage, e.g. also if an employee in the organization leave the job position.

If would be nice from organization to set a default technical contact or bull actions to set/updated this on all projects, e.g. from `/organization/my-organization/projects` 

![image](https://github.com/umbraco/Umbraco.Cloud.Issues/assets/2919859/6a07f7a9-21d1-44a6-b2e4-b529295293d4)

In may be fine the creator of the project is assigned as technical contact, but it would be great if one or more persons always are added as technical contact(s) for projects created under the organization, e.g. a project manager or tech lead.

**Comment by sajumb** [comment]
Thank you for your feature request. It's clear that as organizations scale and the number of projects increases, the need for more streamlined and efficient management of technical contacts becomes more important. Your suggestion to allow organizations to set a default technical contact or to perform bulk actions for setting or updating technical contacts across all projects is hereby noted.

Thanks @bjarnef
